File: Bottom-API/ViewModel/OrderSizeByBatch.cs
using System.Collections.Generic;

namespace Bottom_API.ViewModel
{
    public class OrderSizeByBatch
    {
        public string MO_Seq {get;set;}
        public string Purchase_No {get;set;}
        public string Missing_No {get;set;}
        public string CheckInsert {get;set;}
        public string MO_No {get;set;}
        public string Delivery_No {get;set;}
        public string Material_ID {get;set;}
        public string Material_Name {get;set;}
        public string Model_No {get;set;}
        public string Model_Name {get;set;}
        public string Article {get;set;}
        public string Supplier_ID {get;set;}
        public string Supplier_Name {get;set;}
        public string Subcon_No {get;set;}
        public string Subcon_Name {get;set;}
        public string T3_Supplier {get;set;}
        public string T3_Supplier_Name {get;set;}
        public List<OrderSizeAccumlate> Purchase_Qty {get;set;}
    }
}

File: Bottom-SPA/src/app/_core/_viewmodels/order-size-by-batch.ts
import { OrderSizeAccumlate } from './ordersize-accumlate';
export interface OrderSizeByBatch {
    mO_Seq: string;
    purchase_No: string;
    missing_No: string;
    checkInsert: string;
    delivery_No: string;
    material_ID: string; 
    material_Name: string; 
    model_No: string; 
    model_Name: string;
    mO_No: string;
    article: string;
    supplier_ID: string; 
    supplier_Name: string; 
    subcon_No: string; 
    subcon_Name: string; 
    t3_Supplier: string; 
    t3_Supplier_Name: string; 
    purchase_Qty: OrderSizeAccumlate[];
}